src/libraries/System.Net.Sockets/src/System/Net/Sockets/TCPListener.cs

```diff
@@ -167,6 +167,8 @@ public void Stop()
             _serverSocket = null;
         }
 
+        void IDisposable.Dispose() => Stop();
```

Comment:
According to the API approval https://github.com/dotnet/runtime/issues/63114#issuecomment-1544434338, we should implement this implicitly:

> Adding `IDisposable` seems good, but the consensus was to implicitly/publicly implement the interface, even though that means TcpListener will have two equivalent Stop members.

Reply:
It is not public. Am I right?

Reply:
Sorry, I will make it public...

Reply:
Should I hide `Dispose` method by `[EditorBrowsable(EditorBrowsableState.Never)]` attribute?

Reply:
The API proposal doesn't recommend it and I don't think there is an advantage, so I would say no.